source/faq.txt

```diff
@@ -61,6 +61,21 @@ tasks with the ``MongoClientSettings`` class:
 For more information on the ``MongoClientSettings`` class, see the 
 :java-docs:`API Documentation for MongoClientSettings <apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.html>`.
 
+How do I fix: "javax.net.ssl.SSLHandshakeException: extension (5) should not be presented in certificate_request"?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is a known error that can occur when using the TLS 1.3 protocol with
+specific versions of JDK. Before connecting to your MongoDB instance or
+cluster, you need to update your JDK to one of the following patch
+versions or newer:
```

Comment:
Just for clarity, can we mention that this only applies if they encounter the error when attempting to connect? Otherwise, I'm not sure the phrase "Before connecting to your MongoDB instance" is that helpful.

"If you encounter this error when connecting to MongoDB, update your JDK to one of the following..."